app/gui/tests/language_server.rs

```diff
@@ -64,7 +64,7 @@ wasm_bindgen_test_configure!(run_in_browser);
 async fn ls_text_protocol_test() {
     let _guard = enso_gui::initializer::setup_global_executor();
     let ide = setup_ide().await;
-    let project = ide.current_project();
+    let project = ide.current_project().unwrap();
```

Comment:
don't use unwrap. I know these are tests, just check if its OK and do the test. If its None, fail the test with meaningful error.

Reply:
> don't use unwrap. I know these are tests, just check if its OK and do the test. If its None, fail the test with meaningful error.

Perhaps `expect` method of Option fits here well.

Reply:
Replaced with `expect`